src/SideNavigation/SideNavigation.js

```diff
@@ -159,12 +159,12 @@ SideNavList.propTypes = {
 };
 
 export const SideNavGroup = props => {
-    const { title, children, className, ...rest } = props;
+    const { title, children, className, headerProps, ...rest } = props;
     return (
         <div
             className={`fd-side-nav__group${className ? ' ' + className : ''}`}
             {...rest}>
-            <h1 className='fd-side-nav__title'>{title}</h1>
+            <h1 {...headerProps} className='fd-side-nav__title'>{title}</h1>
```

Comment:
@jbadan had a similar change to a different component and used `titleProps` as the name, which seems to fit a little better since the content of this element is the `title` prop.

Reply:
On an unrelated note, this element should assume the use of `<h1>`.  There should only be one `<h1>` per page and assuming it will exist in the side navigation seems wrong.  I created #249 to address that in a future story.